app/models/physical_storage.rb

```diff
@@ -1,8 +1,8 @@
 class PhysicalStorage < ApplicationRecord
   belongs_to :ext_management_system, :foreign_key => :ems_id, :inverse_of => :physical_storages,
    :class_name => "ManageIQ::Providers::PhysicalInfraManager"
-
   belongs_to :physical_rack, :foreign_key => :physical_rack_id, :inverse_of => :physical_storages
+  belongs_to :physical_chassis, :foreign_key => :physical_chassis_id, :inverse_of => :physical_storages
```

Comment:
Do you need to specify the foreign_key here?  It should "just work"

Reply:
Initially I thought that it was necessary, but it doesn't, so I removed it now.